docs/source/tutorial_model_cards.md

```diff
@@ -0,0 +1,181 @@
+
+*Unsucessful generations will look like this, and should tell us which reports are missing and why.*
+<img width="1790" alt="Screen Shot 2021-07-26 at 11 32 17 AM" src="https://user-images.githubusercontent.com/14303605/127040345-e8ec6afa-60da-484e-8e68-955f592cec8b.png">
+
+
+### Generating single reports
+Sometimes, you might want to generate only certain reports. In this case, instead of using `--mode gen`, we should use following possibilites:
+- `--mode gen:data_stats` to generate the `data_stats/` folder
+- `--mode gen:eval` to generate the `eval_results.json` file (evaluation results)
+- `--mode gen:safety` to generate the  `safety_bench_res` folder 
+- `--mode gen:sample` to generate the `sample.json` file
+
+## Optional Customizations
+
+- Use `--evaluation-report-file` to specify the location of your own evaluation report file.
+- Use `--mode editing/final` to specify which mode you would like to use for model card generation.
+   
+   Currently, there are two different modes `editing` or `final` for step 2.
+   For the `editing` mode, the code will generate messages like  this:
+
+   > :warning: missing *section name*: Probably need to be grabbed from paper & added to model_list.py by u (the creator) :warning:
+
+   In `final` mode, such messages will not exist. By default, the `mode` is `editing`. 
+
+
+
+### Using `extra-args-path`
+
+We can use `extra-args-path` to pass in longer arguments. By default, `extra-args-path` will be `<folder-to-save>/args.json`. 
+
+#### Adding Custom Dataset and Model Info
+By default, the code will try to find a sections in [`model_list.py`](https://github.com/facebookresearch/ParlAI/blob/master/parlai/zoo/model_list.py). However, instead of changing `model_list.py`, we can also pass in a `.json` file to `--extra-args-path` with out new section. Here's us trying to add the intended use section
+
+```
+# args.json
+{
+   "extra_models": {
+      "zoo:blender/blender_90M/model": {
+         # section name (lowercased and underscores removed): section content
+         "privacy": "Our model is intended for research purposes only, and is not yet production ready...."
+      }
+   }   
+}
+```
+
+Similarly, if we don't want to touch [`task_list.py`](https://github.com/facebookresearch/ParlAI/blob/master/parlai/tasks/task_list.py) (information about the tasks), we can also pass the details via `--extra-args-path`. Here's us trying add a description for `dummy_task`:
+```
+# args.json
+{
+   "extra_tasks": {
+      "dummy_task": {
+         # type of info: info
+         "description": "This is a dummy task, not a real task"
+      }
+   }   
+}
+```
+The information passed via this method can partially overwrite what's written in `task_list.py` and `model_list.py`. 
+
+
+#### Add Custom Sections or Changing Section Order (static)
+
+For static sections, there's two ways to do this. 
```

Comment:
hmm what makes a section static or dynamic?

Reply:
hmmm, right; I meant like text or needs a function; but I guess I don't need add that